configs/en-ru-spring-2024.yml

```diff
@@ -0,0 +1,166 @@
+# The initial configuration was generated using:
+# task config-generator -- en ru --name spring-2024
+#
+# The documentation for this config can be found here:
+# https://github.com/mozilla/firefox-translations-training/blob/05963a02792d6e89b2e84a81356aefe69dbcc0d5/taskcluster/configs/config.prod.yml
+experiment:
+  name: spring-2024
+  src: en
+  trg: ru
+  best-model: chrf
+  use-opuscleaner: 'true'
+  opuscleaner-mode: defaults
+  bicleaner:
+    default-threshold: 0.5
+    dataset-thresholds: {}
+  mono-max-sentences-src: 500_000_000
+  mono-max-sentences-trg: 200_000_000
+  spm-sample-size: 10_000_000
+  spm-vocab-size: 32000
+  teacher-ensemble: 2
+  teacher-mode: two-stage
+  pretrained-models: {}
+datasets:
+
+  # Skipped test/devtest datasets:
+  #  - mtedx/valid - variant dataset
+  #  - mtedx/test - variant dataset
+  #  - wmt20/tworefs - variant dataset
+  #  - wmt18/test-ts - variant dataset
+  #  - wmt14/full - variant dataset
+  devtest:
+  - flores_aug-mix_dev
+  - sacrebleu_aug-mix_wmt19
+  - sacrebleu_aug-mix_wmt17
+  - sacrebleu_aug-mix_wmt15
+  - sacrebleu_aug-mix_wmt13
+  test:
+  - flores_devtest
+  - flores_aug-mix_devtest
+  - flores_aug-title_devtest
+  - flores_aug-upper_devtest
+  - flores_aug-typos_devtest
+  - flores_aug-noise_devtest
+  - flores_aug-inline-noise_devtest
+  - sacrebleu_wmt20
+  - sacrebleu_wmt18
+  - sacrebleu_wmt16
+  - sacrebleu_wmt14
+
+  # The training data contains:
+  #   250,048,886 sentences
+  # 
+  # Skipped datasets:
+  #  - opus_CCMatrix/v1 - ignored datasets
+  #  - opus_bible-uedin/v1 - ignored datasets
+  #  - opus_GNOME/v1 - not enough data
+  #  - opus_ELRC-3855-SWPS_University_Soci/v1 - not enough data
+  #  - opus_Ubuntu/v14.10 - not enough data
+  #  - opus_WikiTitles/v3 - not enough data
```

Comment:
WikiTitles has an issue with direction for en-ru/ru-en

Reply:
I added it to the ignore list.